### PR TITLE
Add theme with different soft state colors

### LINKED ISSUE #4160: A theme with different colors for SOFT and HARD states
## Is your feature request related to a problem? Please describe.
Most of the time I would like to ignore services that are in SOFT Critical state. But currently on a first glance they look the same as the ones in HARD Critical state because of the same color.

## Describe the solution you'd like
I would like to be able to create a color theme where a different color is used for SOFT and HARD Critical state and to create a new theme based on the default one that uses it. I would leave the current theme as default. 

## Describe alternatives you've considered
None.

## Additional context
Now:
Screenshot 2020-05-21 at 18 00 04
My (bad) mockup of how a different colors could look like:

### PR DESCRIPTION
**Pull request by gdubicki**
fixes #4160

### CODE REVIEW

File: public/css/icinga/mixins.less [hunk 1]
```diff
@@ -185,12 +185,24 @@
       background-color: @color-warning-handled;
     }
   }
+  &.state-warning-soft {
```
Comment (nilmerg) [inline public/css/icinga/mixins.less:188]: Shouldn't this be `&.state-warning.soft`?

**Comment by nilmerg** [comment]
Are you going to continue with this?

**Comment by gdubicki** [comment]
Yes, I will @nilmerg . Sorry, I was busy with something else. But adding this feature is getting back on my radar so I should do it within the next week or two.

**Comment by gdubicki** [comment]
When I think about it more, @nilmerg , I think that what we really need is a much bigger feature than this small PR. We would like to globally not treating soft critical state as like a hard state.

**Comment by nilmerg** [comment]
> We would like to globally not treating soft critical state as like a hard state.

A critical state should always be a hard state?

**Comment by nilmerg** [comment]
You seem to not want to continue with this, at least not with solving your request with a theme. If you really want to be able to change the state handling, you should head over to [Icinga 2](https://github.com/Icinga/icinga2).

**Pull request closed without merging**